docs/deploy/404.html

```
@@ -0,0 +1,2995 @@
+                                    </a>
+                                    <div class="line-divider"></div>
+                                    <a href="/org/create" onclick="trackAnalyticsEvent('Create_Org_Clicked')">
+                                        <div class="icon-wrapper"><img src="/docs/assets/images/navbar/custom_icons/org-icon.svg"></div> New Organization
+                                    </a>
+                                </div><!-- end content create new menu -->
+                            </div><!-- end dropdown menu create new -->
+                        <div class="dropdown profile-menu show-when-signed-in"
+                                 tabindex="0"
+                                 data-content="User profile and more"
+                                 data-variation="tiny inverted">
+                                <img class="avatar" alt="Avatar" src="">
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)